generators/app/tasks/fileCreators.js

```diff
@@ -5,7 +5,14 @@ const getPackageJsonAttributes = (projectName, projectVersion, repoUrl) => ({
   title: projectName,
   version: projectVersion,
   jest: {
-    collectCoverageFrom: ['src/**/*.{js,jsx,ts,tsx}'],
+    resetMocks: true,
+    collectCoverageFrom: [
```

Comment:
Should we ignore all i18n.ts? Since we mock it, tests never cover them

Reply:
Since we cannot cover that, the only way to cover is to create a `i18n.test.tsx` with the `addResources` mocked, but it adds nothing, is a waste of time to always add the same code mocking something that should be tested by `i18next` lib. That's why we are ignoring them

Reply:
Adding to the previous comment: I forgot we are not ignoring i18n files right now, we changed the way we include translations by adding an `import './i18n';` in the component that creates a i18n file. first to add coverage to them. Second to avoid adding test to them.

Reply:
I know we weren't ignoring them, I was suggesting to do so. I think importing in each component the i18n file is a little uncomfortable if we can just fix this by ignoring the i18n files when calculating the coverage